tests/fixtures/response/app/routes/echo-request-headers.js

```diff
@@ -0,0 +1,17 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  fastboot: Ember.inject.service(),
+
+  model() {
+    return {
+      headerToEchoBack: this.get('fastboot.request.headers').get('x-fastboot-echo')
+    };
+  },
+
+  afterModel(model) {
+    this.get('fastboot.response.headers')
```

Comment:
You could do this in the model, doesn't have to be in `afterModel`, nor do you need to set the model to be anything since you won't be testing the response body